src/Components/Templates/FormTemplate/FormTemplate.tsx

```diff
@@ -43,15 +45,13 @@ export const FormTemplate = ({ label, inputs, buttonLabel, buttonOnClick, linkLa
                             type={item.type}
                             errorLabel={item.errorLabel}
                             marginTop={getInputMargin(index, item.errorLabel)}
+                            caption={item.caption && item.caption}
```

Comment:
Aqui da pra passar só o item.caption msm

```suggestion
                            caption={item.caption}
```